lib/Version.php

```diff
@@ -16,5 +16,5 @@ class Version
     /**
      * Full version number.
      */
-    const VERSION = '5.1.4';
+    public const VERSION = '5.1.4';
```

Comment:
Note: after the content of this PR has been reviewed, I will also need to bump this to 6.0.0 and write a changelog entry.